BuildConfigGen/Program.cs

```diff
@@ -191,7 +192,11 @@ private static void Main2(string task, HashSet<Config.ConfigRecord> targetConfig
 
                 if (config.isNode16)
                 {
-                    WriteNode16PackageJson(taskOutput);
+                    WriteNodePackageJson(taskOutput, "^16.11.39");
```

Comment:
>"^16.11.39") [](http://example.com/codeflow?start=53&length=12)

better to move this into the record so we can use config.NodePackageVersion and avoid repeating for each node version